### PR TITLE
move administrative boundary using non-biodiversity layer

### PR DESCRIPTION
**Pull request by meomancer**
this fixes #https://github.com/kartoza/LEDET_BIMS/issues/159, fixes #https://github.com/kartoza/LEDET_BIMS/issues/148

This PR is about:
- Move administrative boundaries to use non-biodiversity layer
- Init administrative non-biodiversity layer on migrations
- Layer that called based on map level (like previous version)
- With using non-biodiversity layer, we could change url to use local geonode through admin

selection_040
selection_039
selection_038

I think we need boundaries version that styling using stroke and transparent fill? what do you think @gubuntu @NyakudyaA ?

### CODE REVIEW

**Comment by gubuntu** [comment]
@NyakudyaA please style these layers asap (I don't have permission)

@meomancer this doesn't help with boundary-level clustering though. For that you could refactor your existing boundary models to route to the boundary tables in the GeoNode data. 

You can also fetch GeoJSON from GeoNode/GeoServer WFS if you need it in the client.

**Comment by meomancer** [comment]
```
@meomancer this doesn't help with boundary-level clustering though. For that you could refactor your existing boundary models to route to the boundary tables in the GeoNode data.
```
how do i get the boundary table in geonode data @gubuntu ? does it exist? How do i try on my local geonode?

```
You can also fetch GeoJSON from GeoNode/GeoServer WFS if you need it in the client.
```
We are using wms (in this PR) because the geojson of boundary is quite big and taking time to download it.

File: bims/static/js/views/olmap_layers.js [hunk 1]
```diff
@@ -96,18 +87,55 @@ define(['shared', 'backbone', 'underscore', 'jquery', 'ol', 'views/layer_style']
...
+            if ($checkbox.length == 0) {
+                return true
+            }
+            return $checkbox.is(':checked');
+        },
+        changeLayerAdministrative: function (administrative) {
+            var self = this;
+            switch (administrative) {
+                case 'province':
+                    self.currentAdministrativeLayer = self.administrativeLayersName[0];
```
Comment (dimasciput) [inline bims/static/js/views/olmap_layers.js]: @meomancer Why don't use dictionary instead?

**Comment by dimasciput** [comment]
@meomancer please also update the compiled js

**Comment by meomancer** [comment]
Hi @NyakudyaA 
I cant access to ibims now, need authorization. Is it a must? because i want to fix this with accessing to ibims. (i can't access capabilities too)
- geonode:district_municipalities_2011
- geonode:district_council_sa_2011
- geonode:provinces_sa_2011

**Comment by NyakudyaA** [comment]
@meomancer you should be able to access all the layers now.
geonode:district_municipalities_2011 and geonode:district_council_sa_2011 are identical so I am going to delete the geonode:district_council_sa_2011 from Geonode

**Comment by gubuntu** [comment]
@meomancer, @NyakudyaA the geonode/geoserver services should be using autherntication with usernames shared from BIMs. @NyakudyaA there should be a 'system' username that application calls can use e.g. when the system fetches admin boundaries. 

@meomancer as per slack, you can also fetch those boundaries using your existing django models, just adapted and routed to the gis database.

**Comment by NyakudyaA** [comment]
@gubuntu  cc @meomancer  in order to use the geonode permissions you maybe need to access the service directly from geonode ows rather than geoserver wfs

**Comment by gubuntu** [comment]
That's correct @NyakudyaA. Direct access to geoserver url should be blocked to everyone except admins. 

BIMS system and users must use the GeoNode url so requests get proxied through. 

@NyakudyaA can you confirm it is the GeoNode url that gets advertised on csw and the various GetCapabilities?

**Comment by meomancer** [comment]
hi @gubuntu @NyakudyaA 
could you explain me how to do this?

> @meomancer as per slack, you can also fetch those boundaries using your existing django models, just adapted and routed to the gis database.

because currently no data on geonode on my local

**Comment by gubuntu** [comment]
@meomancer you need to restore the GeoNode+Geoserver backup onto your dev machine that @NyakudyaA and @dimasciput have been trying to restore and get to work on testing.

**Comment by dimasciput** [comment]
@meomancer can you update this? then try to use the layer from testing.limpopobims.

**Comment by gubuntu** [comment]
@meomancer is this deployed on testing?

Are the cluster boundaries what is called the 'Administrative' layer in the map?

On testing it still draws under the basemap and dragging it above the biodiversity layer does nothing. And its labels conflict with the basemap labels. 

What is holding this up?

**Comment by dimasciput** [comment]
I will merging this one and use layers from lbims since we still can't access the testing layers.

**Comment by gubuntu** [comment]
@dimasciput @NyakudyaA I thought that was fixed, why can't BIMS access local geonode layers yet?

**Pull request merged**